package.json

```diff
@@ -40,7 +40,8 @@
     "test:coverage": "npm test -- --coverage",
     "test:watch": "npm test -- --watch",
     "test:update": "npm test -- -u",
-    "commit": "npm install npm run format && npm run lint:fix && npm test -- -u && npm run test:coverage",
```

Comment:
I think this line wasn't working (`&&` was missing after `npm install`). So I just removed it. `npm fix-all` should be similar.